src/Application/src/CalculatorApplication.cpp

```diff
@@ -7,22 +7,41 @@
  */
 ////////////////////////////////////////////////////////////////////////////
 
+#include "CalculatorApplication.h"
+
 #include <algorithm>
 #include <cmath>
+#include <fstream>
 #include <future>
 #include <iostream>
 
-#include "CalculatorApplication.h"
+#include "flatbuffers/flatbuffers.h"
+
 #include "CalculatorStrings.h"
+#include "Color.h"
 #include "CompoundExpressionCalculator.h"
 #include "CustomCalculatorFactory.h"
+#include "History.h"
 #include "SimpleExpressionCalculatorFactory.h"
 
 namespace calculator
 {
+    //place to test stuff please ignore
+    void CalculatorApplication::playground()
+    {
+        int i = 10;
```

Comment:
why 10? seems a magic number - explain in comment
Better name that i

Reply:
//place to test stuff please ignore :)